_data/tools.yml

```diff
@@ -17,5 +17,4 @@ toc:
   - title: Helm
     path: https://github.com/kubernetes/helm
   - title: Kompose
-    path: https://github.com/kubernetes-incubator/kompose
-  
+    path: /docs/tools/kompose/
```

Comment:
I think I know what's causing the problem here. Due to some recent GitHub Pages changes, we no longer specify title here. Please see this example for how to specify titles: [_data/guides.yml](https://github.com/kubernetes/kubernetes.github.io/blob/master/_data/guides.yml#L8) and [docs/whatisk8s.md](https://github.com/kubernetes/kubernetes.github.io/blame/master/docs/whatisk8s.md#L5)

Basically remove `title: Kompose` here and insert title in docs/tools/kompose/index.md